File: src/main/java/com/fifo/ticketing/domain/performance/mapper/PlaceMapper.java
package com.fifo.ticketing.domain.performance.mapper;

import com.fifo.ticketing.domain.performance.dto.PlaceResponseDto;
import com.fifo.ticketing.domain.performance.entity.Place;

public class PlaceMapper {

    public static PlaceResponseDto toDtoForPerformanceCreate(Place place) {
        return new PlaceResponseDto(place.getId(), place.getName());
    }
}

File: src/main/java/com/fifo/ticketing/domain/performance/mapper/PlaceMapper.kt
package com.fifo.ticketing.domain.performance.mapper

import com.fifo.ticketing.domain.performance.dto.PlaceResponseDto
import com.fifo.ticketing.domain.performance.entity.Place

object PlaceMapper {
    @JvmStatic
    fun toDtoForPerformanceCreate(place: Place) = PlaceResponseDto(place.id, place.name)
}
